MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include LICENSE.txt CHANGELOG.txt MANIFEST.in requirements_docs.txt requirements_tests.txt requirements.txt
```

Comment:
Maybe `include *.txt` is a better option since some packages might have other txt  files they want to include

Reply:
But this is also good!

Reply:
That would be easier, but it could also grab files that were not intended to be included?

I grabbed the MANIFEST.in from a few of our sub-packages and they all that format. I don't have a good solution, just yet, but wanted to raise it so we come up with a standard for the sub-packages.

Going to mark this as WIP to pull in other folks so we can come up with a collective answer.

Reply:
I think the explicit list is best here. Since it's supposed to be a template, this `MANIFEST.in` shows how to include only the text files that will be in a minimally-specified package. Any additional thing you can bring along by adding to the list, like is done in other areas (e.g. the setup.py requirements list).